src/components/Person/Person.jsx

```diff
@@ -1 +1,47 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerSex = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${partnerSex}`
+          : 'I am not married'
+}
+      </p>
+    </section>
+
+  );
+};
+
+Person.defaultProp = {
+  name: null,
+  partnerName: null,
+};
```

Comment:
There is no need to set defaultProp for .isRequired elements. You should write here the default values for `partnerName` and `age` (because they could be not passed)  and not write `isRequired` for these values in Person.propTypes.